numpy/core/tests/test_half.py

```diff
@@ -266,17 +266,17 @@ def test_half_correctness(self):
         if len(a32_fail) != 0:
             bad_index = a32_fail[0]
             assert_equal(self.finite_f32, a_manual,
-                 "First non-equal is half value %x -> %g != %g" %
-                            (self.finite_f16[bad_index],
+                 "First non-equal is half value 0x%x -> %g != %g" %
```

Comment:
Unrelated?

Reply:
Uhh, yeah, the test had failed on me when I had a buggy version which caused this to fail cryptically...